src/components/search/_search.scss

```diff
@@ -279,15 +279,20 @@
     fill: $icon-01;
     border: 2px solid transparent;
     border-left: 0;
+  }
 
-    &:hover {
-      background-color: $hover-ui;
-      border-bottom: 1px solid $ui-04;
-    }
-    &:active {
-      @include focus-outline('outline');
-      background-color: $selected-ui;
-    }
+  .#{$prefix}--search-close:hover {
+    background-color: $hover-ui;
+    border-bottom: 1px solid $ui-04;
+  }
+  .#{$prefix}--search-close:active {
+    @include focus-outline('outline');
+    background-color: $selected-ui;
+  }
+
+  .#{$prefix}--search--xl .#{$prefix}--search-close svg {
+    width: 20px;
```

Comment:
Should we use `rem` or `mini-units` helpers for this?

Reply:
I removed it and will add the icon in react